deepdrivemd/aggregation/stream/aggregator.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+from deepdrivemd.aggregation.stream.config import StreamAggregation
+from deepdrivemd.utils import Timer, timer, parse_args, intarray2hash
+import time
+import os
+import sys
+import adios2
+import math
+import queue
+import subprocess
+import itertools
+from deepdrivemd.data.stream.adios_utils import AdiosStreamStepRW
+from typing import List, Dict, Tuple
+from pathlib import Path
+
+
+def find_input(cfg: StreamAggregation) -> List[Path]:
```

Comment:
This function says it returns List[Path] but it actually returns List[str].

It may be good to update the code to use Path objects instead of strings, since path objects have wonderful helper functions for dealing with path parsing, and other os.path functions. I'll let you decide though, we just need the type hints to be the same as the actual types.

Reply:
Changed to List[str]